apps/ledger-live-mobile/src/dynamicContent/useDynamicContentLogic.ts

```diff
@@ -46,6 +50,12 @@ export const useDynamicContentLogic = () => {
       .map(card => mapAsLearnContentCard(card))
       .sort(compareCards);
 
+    const categoriesCards = filterByType(mobileContentCards, ContentCardsType.category)
+      .map(card => mapAsCategoryContentCard(card))
+      .sort(compareCards);
+
+    dispatch(setDynamicContentCategoriesCards(categoriesCards));
+    dispatch(setDynamicContentMobileCards(mobileContentCards));
     dispatch(setDynamicContentWalletCards(walletCards));
     dispatch(setDynamicContentAssetsCards(assetCards));
     dispatch(setDynamicContentNotificationCards(notificationCards));
```

Comment:
We have too much dispatch there + we should avoid to parse the data before storing it in the Redux Store. The flow should be:
`Fetch -> Dispatch -> Store -> Selector -> View`
Here we already prepare the data for our views or logic but it creates complexity if we want to do something else with the data we got from the API.

Reply:
We'll implement this change in another PR as it implies some modifications on the existing content cards system